recipes/auspice/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version = "2.47.0" %}
+{% set version = "2.48.0" %}
 
 package:
   name: auspice
   version: {{ version }}
 
 source:
   url: https://registry.npmjs.org/auspice/-/auspice-{{ version }}.tgz
-  sha256: c43c570d490c56eead5a91174bc8f782d377bae6d33f12122a225ede2fef9b76
+  sha256: 79f88345fde686890e36919338a720807bb829b9f9edec586efc89c5c154e38a
 
 build:
   number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```

Comment:
```suggestion
    - {{ pin_subpackage("auspice", max_pin="x.x") }}
```

Reply:
Thanks @mencian! I clearly missed this from the instructions 🤦

Reply:
No problem, run_exports was added _very_ recently.